pyproject.toml

```diff
@@ -85,7 +85,7 @@ norecursedirs = [
     "build",
     ".tox"
 ]
-testpaths = ["tests/*.py", "tests/blocks/*.py", "tests/integration/*.py"]
+testpaths = ["tests/*.py", "tests/blocks/*.py", "tests/integration/*.py", "tests/unit/*.py"]
```

Comment:
Would probably be better to just test everything under `tests/`? 
https://github.com/brightway-lca/brightway2-data/commit/2e0f53867a6da8bb9beff454ea4a2ce30c54f844